.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "steelbrain",
+  "rules": {
+    "no-param-reassign": 0,
```

Comment:
Btw, makes things clearer (for us humans, no difference to `eslint` 😛) if you use the `'off'` designation here instead of `0`.